apps/frontend/src/main.tsx

```diff
@@ -3,17 +3,18 @@ import ReactDOM from 'react-dom/client';
 import { QueryClient, QueryClientProvider } from 'react-query';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import MapPage from './pages/mapPage/MapPage';
+import VolunteerPage from './pages/volunteerPage/VolunteerPage';
 
 const queryClient = new QueryClient();
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
     <QueryClientProvider client={queryClient}>
-        <BrowserRouter>
-          <Routes>
-            <Route path="/" element={<MapPage />} />
-          </Routes>
-        </BrowserRouter>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<MapPage />} />
```

Comment:
Could you try adding this as a route: `<Route path="/volunteer_page" element={<VolunteerPage />} />`

Reply:
pushed changes for both comments; for Resources, I assumed you meant substituting BoxPanel with FlippableTile, since Resources represents a grid of tiles?

Reply:
Yes, that's right! Sorry